### DISCUSSION TITLE
Starknet AirDrop doesn't fit by mistake

### DISCUSSION THREAD

**Discussion by zomand** [starknet-io/.github#3] [General]
### **I propose to discuss the error in STRK token allocation in this thread**

You can check your stats and balance on this site https://coinstats.app

![chart](https://github.com/starknet-io/.github/assets/103496929/db6ab227-3c18-475d-901b-ab2f0d87c37c)
![image (3)](https://github.com/starknet-io/.github/assets/103496929/f8ebb769-6442-47cc-81dd-584b8d885a50)

Hi, I'm surprised that my account didn't qualify for STRK tokens.

 The account has been active since late 2022, when I first learned about this project online.

I assume that the account was mistakenly not included in the AirDrop pickup list.

I attach screenshots that indicate that at the time of the photo on the account were funds, as well as the account was suitable for other criteria.

It is a shame that the project, in which invested so much effort, missed my investment in it without attention. I tried so hard to attract the attention of the whole internet to your project, publishing posts and art in social networks.

Please check my account. Perhaps there was a mistake somewhere.

My wallet address
0x00c7DBF862Edb5d23acDf22570e98eb725CE5902E89BcCA12fBa7E158e318f48

**Comment by sinopsis08** [comment]
а еще проверьте сотни тысяч аккаунтов, не возможно. а у вас точно ошибка

**Reply by Danialz25** [reply]
![Screenshot 2024-02-18 115607](https://github.com/starknet-io/.github/assets/97152864/fc57c99e-6a3d-4f0c-8b26-8f9f9f9cd944)


i always hold up to 0.005 but why my allocation 0?

my address : 0x07FeE0E12611c6a8D5bCCA1E9f295828814280c360daeFef70477e4FADba2940

**Reply by S3T4RC0S** [reply]
I don’t receive

**Comment by EldePan** [comment]
My wallet address too
![stk1](https://github.com/starknet-io/.github/assets/127691840/12d99742-ca4d-4367-aef2-c30136e4d41f)

0x0193f3ce36ba20e639c722e658bb2ce84701ca8ea3afbfbba2ab10e9ff56237b

**Comment by Tonybostonn** [comment]


**Reply by zomand** [reply]
you can check your stats on this site 


https://coinstats.app/

**Comment by Noob4ik0** [comment]
![Снимок экрана (223)](https://github.com/starknet-io/.github/assets/133154927/b80ed41b-418e-4806-8951-04ab0710fb2f)
0x0790db22Fd852BA4f92b3cEf3C0690Bd334F0007B9C0962FAE9743e3d9DE2702 hallo,its also problem,yuo can check again

**Comment by dadyatolya** [comment]
my wallet adress I not e-bagger 0x05e4E60Cfb2D2301e9f0dAD6EEeC1DB4b07D21B3dC1750224045ed79C308751C

**Comment by Alexander-19910206** [comment]
Good afternoon! My wallet fits all the parameters and at the time of the photo it was 0.017eth, please check again! I hope for a fair decision!
![Знімок екрана 2024-02-16 211330](https://github.com/starknet-io/.github/assets/133955386/ac8b4c83-d39c-44b3-b5d8-641a18f259f2)
0x07781f48D0d440c8187a2a00Eed00f7196B093BF26414BC2ee00Fd09998384EA

**Comment by anglosax770** [comment]
0x018b4f1811455bcddc4735b48734bfe94aa2bd1be767bf9b7e0995b373a4e42b
![image](https://github.com/starknet-io/.github/assets/125607008/41d89b04-3da1-47f8-8b45-5699ab80e150)
![image](https://github.com/starknet-io/.github/assets/125607008/8e33111b-cd4a-48c5-9976-d7f32bcccd1a)

**Comment by totalmenroket** [comment]
please check again my balance and the criteria are 100 percent suitable at the time of the photo
!
![Знімок екрана 2024-02-16 233650](https://github.com/starknet-io/.github/assets/139012654/f1011035-b3c0-441f-b314-300d0ad6e5f2)

0x03031AB0Cd0D26a1c5Ae3436e4aCE5BC0515d7565E0FEa30E5109DEC2747C394

**Comment by Saran5553** [comment]
Me too I have satisfy all the four criteria...still am not eligible how it possible pls check my address 👇

0x031109e2b141cc8dd16a8add2484231aa4a2ccaac5bf321f4220262db16ca3f1
![Screenshot_205](https://github.com/starknet-io/.github/assets/86559347/91cfb898-daef-45a6-ad7c-99bc1f1d495a)

**Comment by feandrrrr** [comment]
![Screenshot_20240217-023028](https://github.com/starknet-io/.github/assets/131308541/427340fa-39ab-4daa-96fa-7ab4e22ec9ec)
0x06be293c9f93389457b09b304a90de459ac97f0f48d7aeaf201e402197a98ac7
Lol, 0 allocation

**Comment by kotukmeow** [comment]
0x06D8EDb78a93c04De99874eA307F8b766826A2ba27b4498Dfc1fBE5864723180
![1](https://github.com/starknet-io/.github/assets/128662534/3c3666fc-2d6d-460e-b443-c27b36d37096)


my balance as of November 15 over 0.01 in eth
30+ txs
My unique months are March, May, August, September, October

volume 100% more than 100 dollars
Hash https://starkscan.co/tx/0x012e5bea7d89363ad911c038c74a416374a142e339ffb55002797d1577908f9f
(swap 500$)

**Comment by Gryhail** [comment]
Dear Team,

I have been very saddened to see my wallet being not eligible for STARK allocation.
I have been participating in the Starknet ecosystem since 2022, tried my best to try out every dapp, even during the earliest buggy stages, minted and sold NFTs, swapped significant amount of money. During all that time I have been running both Starknet and Madara nodes, as well.

With few tools available to me now, such is coinstats.app, I see that on November 15, my wallet balance was 0.01645 eth, which seems to be enough to be included in the drop.
![Coinstats](https://github.com/starknet-io/.github/assets/82543741/43b4b439-b4d8-403d-bdee-993b965ccfea)

My wallet address: 0x03dD1c646d0070D32F8574112F427E2Fc967C6A9bf39bc96bceff1b8C0d14A33
I'll be extremely overjoyed if the team reconsidered and included such an early bird as me too.
